seat.c

```diff
@@ -568,7 +584,10 @@ handle_cursor_motion(struct wl_listener *listener, void *data)
 static void
 drag_icon_damage(struct cg_drag_icon *drag_icon)
 {
-	output_damage_drag_icon(drag_icon->seat->server->output, drag_icon);
+	struct cg_output *output;
+	wl_list_for_each(output, &drag_icon->seat->server->outputs, link) {
+		output_damage_drag_icon(output, drag_icon);
```

Comment:
Why is it necessary to do this on every output?

Reply:
A surface can be visible on multiple outputs at once, so we must report damage to them all. This is in line with what sway does.

Reply:
Of course, makes sense :sweat_smile: